src/components/views/User/Edit.tsx

```diff
@@ -516,7 +514,7 @@ class UserEdit extends React.Component<IProps, IState> {
         const currentOAuthConnections =
             this.state.newOAuthConnections ?? this.state.user?.oAuthConnections;
         if (
-            this.state.user?.name.toLowerCase() === "admin" || // admin user can't have OAuthConnections
+            this.state.user?.name!.toLowerCase() === "admin" || // admin user can't have OAuthConnections
```

Comment:
Why the null assert? This can, in fact be undefined if user is undefined
```suggestion
            this.state.user?.name.toLowerCase() === "admin" || // admin user can't have OAuthConnections
```